### PR TITLE
feat(service-portal): Fullscreen inbox pdf

### PR DESCRIPTION
**Pull request by thordurhhh**
## What

Add fullscreen option to PDF documents in inbox.

## Why

Better usability for documents.

## Screenshots / Gifs

Fullscreen:
Screenshot 2024-01-26 at 09 58 03

On page before expanding:
Screenshot 2024-01-26 at 09 57 50

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/service-portal/documents/src/components/DocumentRenderer/DocumentRenderer.css.ts [hunk 1]
```diff
@@ -14,11 +13,40 @@ export const pdfControls = style({
...
+globalStyle(`${pdfControls} button`, {
+  boxShadow: 'none',
+})
+
+globalStyle(`${pdfAction} + button`, {
+  '@media': {
+    [`(max-width: ${theme.breakpoints.md}px)`]: {
+      display: 'none',
+    },
+  },
```
Comment (thorkellmani) [inline libs/service-portal/documents/src/components/DocumentRenderer/DocumentRenderer.css.ts:51]: Cool, but why not use .

```typescript
...themeUtils.responsiveStyle({
    md: {
       display: none,
       // or 
       justifyContent: center
  }),
```

File: libs/service-portal/documents/src/components/DocumentRenderer/PdfDocument.tsx [hunk 2]
```diff
@@ -1,60 +1,104 @@
...
+    }
+
+    // Size of PDF Canvas
+  }, [ref.current?.querySelectorAll('canvas')?.[0]?.width])
+
   return (
     <>
       <Box
         className={styles.pdfControls}
+        // justifyContent={'spaceBetween'}
```
Comment (thorkellmani) [inline libs/service-portal/documents/src/components/DocumentRenderer/PdfDocument.tsx:42]: remove?

**Review by thorkellmani — APPROVED** [review]
LGTM 🐦 , 2 comments tho

File: libs/service-portal/documents/src/components/DocumentRenderer/DocumentRenderer.css.ts (hunk 1 above)
Comment (thordurhhh) [reply libs/service-portal/documents/src/components/DocumentRenderer/DocumentRenderer.css.ts:51]: 👴

**Pull request merged**